packages/default-storage-expo-plugin/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "expo-module-scripts/tsconfig.plugin",
+  "compilerOptions": {
+    "outDir": "build",
+    "rootDir": "src"
+  },
+  "include": ["./src"],
+  "exclude": ["**/__mocks__/*", "**/__tests__/*"]
```

Comment:
nit: u don't have mocks nor tests, so u can remove this

Reply:
I left this in to prevent any future mishaps if/when tests were added.